kubernetes/core/src/main/kotlin/com/google/kubernetes/tools/core/KubectlUiResources.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
suggestion:

instead of creating a new bundle, you can just reuse CoreBond. See this comment for relevant discussion: https://github.com/GoogleContainerTools/google-container-tools-intellij/pull/119#discussion_r246060948